src/App.tsx

```diff
@@ -2,13 +2,27 @@ import React from 'react';
 
 import './App.scss';
 
-// import posts from './api/posts';
-// import comments from './api/comments';
-// import users from './api/users';
+import posts from './api/posts';
+import comments from './api/comments';
+import users from './api/users';
+
+import { Comment } from './Types/Comment';
+import { Post } from './Types/Post';
+import { User } from './Types/User';
+
+import { PostList } from './components/PostList/PostList';
+
+const preparedPosts: Post[] = posts.map((post) => ({
+  ...post,
+  user: users.find((user: User) => user.id === post.userId) || null,
+  comments: comments
+    .filter((comment: Comment) => comment.postId === post.id) || null,
```

Comment:
filter always return array

```suggestion
  comments: comments
    .filter((comment: Comment) => comment.postId === post.id),
```